app/helpers/logging_helper.rb

```diff
@@ -0,0 +1,34 @@
+module LoggingHelper
```

Comment:
Perhaps instead of using a module you should use a class. The way you're using it isn't very module like. Make it a class and put in lib would be my suggestion.